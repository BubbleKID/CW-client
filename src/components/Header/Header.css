.header-container {
    background-color: #0b0d12;
    border-bottom: solid 1px rgba(255,255,255,0.15);
}

.header {
    height: 60px;
    margin: 0 auto;
    max-width: 1600px;
    display: flex;
    justify-content: space-between;
    align-items: baseline;
}

.header-items {
    height: 60px;
    font-size: 16px;
}

.header-items a {
    position: relative;
    display: inline-block;
    height: 60px;
    margin-right: 30px;
    text-transform: uppercase;
}

.header-button {
    display: inline-block;
}

.header-button span {
    cursor: pointer;
    display: inline-block;
    line-height: 60px;
}

.header-logo {
    cursor: pointer;
    height: 60px;
    margin-right: 30px;
    fill: #009ae4;
}

.header-logo svg{
    line-height: 22px;
    position: relative;
    top: 50%;
    transform: translateY(-50%);
    height: 34px;
}

.profile a {
    cursor: pointer;
    font-size: 17px;
    text-transform: capitalize;
    margin-right: 30px;
}