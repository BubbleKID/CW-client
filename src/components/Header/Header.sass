.header 
    height: 60px
    margin-bottom: 60px
    background-color: #0052a0
    padding: 20px 0

    &__container
        max-width: 1200px
        margin: auto

        & img 
            height: 75px

