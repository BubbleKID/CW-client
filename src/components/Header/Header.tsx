import React from 'react';
import './Header.sass';
import  Logo1 from '../../images/CWH_logo_1.png';
import  Logo2 from '../../images/CWH_logo_2.png';

export default function Header() {
    return (
        <div className="header">
            <div className="header__container">
                <a className="header__logo"><img src={Logo1} alt="logo1" /></a>
                <a className="header__logo"><img src={Logo2} alt="logo2" /></a>
            </div>
        </div>
    )
}