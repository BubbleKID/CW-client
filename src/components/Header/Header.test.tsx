import React from 'react';
import TestRenderer  from 'react-test-renderer';
import Header from './Header';

test('Header should be rendered', () => {
    const component = TestRenderer.create(<Header/>,);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});