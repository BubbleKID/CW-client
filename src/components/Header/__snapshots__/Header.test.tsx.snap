// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should be rendered 1`] = `
<div
  className="header"
>
  <div
    className="header__container"
  >
    <a
      className="header__logo"
    >
      <img
        alt="logo1"
        src=""
      />
    </a>
    <a
      className="header__logo"
    >
      <img
        alt="logo2"
        src=""
      />
    </a>
  </div>
</div>
`;
